docs/keypoints_convention.md

```diff
@@ -1,52 +1,68 @@
-## Keypoints convention
+# Keypoints convention
 
-### Overview
+## Overview
 
 Our convention tries to consolidate the different keypoints definition across various
 commonly used datasets. Due to differences in data-labelling procedures, keypoints across datasets with the same name might not map to semantically similar locations on the human body. Conversely, keypoints with different names might correspond to the same location on the human body. To unify the different keypoints correspondences across datasets, we adopted the `human_data` convention
 as the base convention for converting and storing our keypoints.
 
-### How to use
+## How to use
 
-#### Converting between conventions
+### Converting between conventions
 
 Keypoints can be converted between different conventions easily using the `convert_kps` function.
 
-To convert a `agora` keypoints to `coco` convention, specify the source and
+To convert a `human_data` keypoints to `coco` convention, specify the source and
 destination convention for conversion.
 
 ```python
 from mmhuman3d.core.conventions.keypoints_mapping import convert_kps
 
-keypoints_agora = np.zeros((100, 127, 3))
-keypoints_coco, mask = convert_kps(keypoints_agora, src='agora', dst='coco')
+keypoints_human_data = np.zeros((100, 190, 3))
+keypoints_coco, mask = convert_kps(keypoints_human_data, src='human_data', dst='coco')
+assert mask.all()==1
 ```
 
 The output `mask` should be all ones if the `dst` convention is the subset of the `src` convention.
 You can use the mask as the confidence of the keypoints since those keypoints with no correspondence are set to a default value with 0 confidence.
 
 
-#### Converting with mask
+### Converting with mask
 
 If you have occlusion information of your keypoints, you can use an original mask to mark it, then the information will be updated into the returned mask.
-E.g., you want to convert a `smpl` keypoints to `coco` keypoints, and you know its `left_hip` is occluded. You want to carry forward this information during the converting. So you can set an original_mask and convert it to `coco` by doing:
+E.g., you want to convert a `smpl` keypoints to `coco` keypoints, and you know its `left_shoulder` is occluded. You want to carry forward this information during the converting. So you can set an original_mask and convert it to `coco` by doing:
```

Comment:
Do we design `mask` to play the role of occlusion indication? I suppose this may be the purpose of `confidence`? Because occlusion estimation often uses a `confidence` to indicate how sure the network thinks a keypoint is invisible. Maybe we can use a simpler example just to showcase the `convert_kps`?